tensorboard/components/experimental/plugin_lib/test/test.ts

```diff
@@ -89,4 +91,67 @@ describe('plugin lib integration', () => {
       });
     });
   });
+
+  describe('lib.core', () => {
+    describe('#getURLPluginData', () => {
+      /**
+       * These tests use tf_globals' fake hash to make tf_storage think that the
+       * host's URL has been updated.
+       */
+      it('returns URL data', async function() {
```

Comment:
let's write one more spec. 
```ts
tf_globals.setFakeHash('sample_plugin&p.sample_plugin&p.sample_plugin.');
```

(user handcrafted the hash but forgot to add something after the period)

Reply:
Done.